Kitodo/src/test/java/org/kitodo/production/services/dataformat/MetsServiceIT.java

```diff
@@ -71,4 +78,38 @@ public void testReadXML() throws Exception {
                 "131", "132", "133", "134", "uncounted", "uncounted", "uncounted"),
             workpiece.getMediaUnits().stream().map(MediaUnit::getOrderlabel).collect(Collectors.toList()));
     }
+
```

Comment:
Seems to be formatted incorrectly

Reply:
Nope, that's right.

Reply:
Mhm... my formatter is reformatting it :(

Reply:
Mine not. But we know that they do not agree 100%.